web-client/src/views/IdleActivityMonitor.tsx

```diff
@@ -46,6 +46,9 @@ export const IdleActivityMonitor = connect(
         handleIdleLogoutSequence();
       }, 1000);
 
+      // Make sure we broadcast activity as soon as a new tab loads
+      broadcastIdleStatusActiveSequence();
```

Comment:
Otherwise, a user could sign in, refresh, do nothing for 55 minutes, and never see the "Are you still there?" modal.